src/main/markdown/doc/latest/polymer-tutorial/introduction.md

```diff
@@ -0,0 +1,48 @@
+# Building a modern GWT app using Polymer Elements:
+
+## Introduction
+
+In this tutorial, you’ll learn how to write a **TodoList** application using [Polymer][0] [Web Components][1]. Web Components define a collection of standards allowing us to bundle markup and styles into custom HTML elements. Polymer is a JS library to make easier deal with Web Components.
+
+In this case, we want to fulfil the [Material Design][2] specification. Therefore, we’ll be using the Vaadin [gwt-polymer-elements][3] library. This library, generated by [gwt-api-generator][4], is a JsInterop wrapper for the Polymer [Iron and Paper][5] elements.
+
+Before we start, you might want to try out the [TodoList][6] application to see what we are about to build.
+
+## **Disclaimer**:
```

Comment:
The disclaimer should be something brief, standard and well designed then (some special css), and we must put it in many sites apart from this tutorial pages, like in those pages speaking about ant vs maven, php vs ruby, etc.